### PR TITLE
FINERACT-822 Enabling ErrorProne error checks

### PR DESCRIPTION
**Pull request by percyashu**
FINERACT-822

### CODE REVIEW

**Comment by percyashu** [comment]
@xurror @awasum @vorburger LGTY?

**Review by awasum — APPROVED** [review]
LGTM. Forge ahead. As we discussed lets get everything Error Prone done within the next few working days. You can raise multiple PRs using separate branches. Good work @percyashu

File: fineract-provider/src/main/java/org/apache/fineract/portfolio/savings/service/SavingsEnumerations.java [hunk 1]
```diff
@@ -200,6 +199,7 @@ public static SavingsAccountTransactionEnumData transactionType(final SavingsAcc
                 optionData = new SavingsAccountTransactionEnumData(SavingsAccountTransactionType.AMOUNT_RELEASE.getValue().longValue(),
                         SavingsAccountTransactionType.AMOUNT_RELEASE.getCode(), "Release Amount");
                 break;
+            default:
```
Comment (vorburger) [inline fineract-provider/src/main/java/org/apache/fineract/portfolio/savings/service/SavingsEnumerations.java:202]: @percyashu @awasum I disagree with this, this seems wrong to me - I've raised new #997 for it.

PS: I'm happy to help code review such PRs, if you would like me to.

Comment (awasum) [reply fineract-provider/src/main/java/org/apache/fineract/portfolio/savings/service/SavingsEnumerations.java:202]: @vorburger . That is Ok with me to review such PRs

I am looking into the points you raised wrt this merged PR.

**Pull request merged**